apps/api/src/common/graphql/services/core.service.ts

```diff
@@ -23,7 +23,10 @@ export abstract class CoreService<TEntity> {
   ): Promise<{ items: TEntity[]; total: number }> {
     this.logger.log(`Getting all ${alias} with options`);
 
-    const queryBuilder = this.repository.createQueryBuilder(alias);
+    // Left Join and select is used to fetch and diaplay related applicationDetails in response
+    const queryBuilder = this.repository
+      .createQueryBuilder(alias)
+      .leftJoinAndSelect(`${alias}.applicationDetails`, 'notify_applications');
```

Comment:
This might break if we pass an `alias` value other than `notification`. So we can add an if check to only perform this join in case that alias is passed.

Also, small type in the comment above, fix 'diaplay' -> 'display'

Reply:
Fixed this